packages/arb-token-bridge-ui/src/pages/api/eth-deposits-custom-destination.ts

```diff
@@ -0,0 +1,154 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import { gql } from '@apollo/client'
+
+import {
+  getL1SubgraphClient,
+  getSourceFromSubgraphClient
+} from '../../api-utils/ServerSubgraphUtils'
+import { FetchEthDepositsToCustomDestinationFromSubgraphResult } from '../../util/deposits/fetchEthDepositsToCustomDestinationFromSubgraph'
+
+type NextApiRequestWithDepositParams = NextApiRequest & {
```

Comment:
For future: we should abstract and reuse this type across all our APIs

Reply:
I will create a ticket